### DISCUSSION TITLE
(Find someone to) Provide linedrawings / technical drawings for our patterns

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#1625] [Forum]
We have a React component that provides the line drawings of our patterns, but it's getting increasingly tricky to maintain it because it merely lists individual paths, which is a severe limit on what we can do.
Furthermore, many of the drawings are done by Joost and that guy can't draw to save his life.

So, I'd like to make an effort to find someone who can do linedrawings / technical drawings, and would be willing to (over time) do all of them.

If we can't find someone in the community, we could perhaps look elsewhere.

To be clear: I am talking about this kind of stuff:

![Drawing](https://images.squarespace-cdn.com/content/v1/56eaefe3d210b8c5e875c0fc/1592911186674-E75HO1ZUQTFCSZBRZQG4/lydia_claxton__technical_drawing_award+%28dragged%29.jpg?format=1000w)

Tagging this with v3 because I feel it would go nice with our redesign

Perhaps some things to keep in mind about the technical requirements for such drawings:

 - Can we haz SVG?
 - viewBox should probably be the same for all of them
 - Line widths & color should be consistent
 - Best case scenario, these images are themeable (by only use `currentColor` & transparancy to get a monochrome image)

But I'm happy to take advice from illustrators on this

**Comment by darigovresearch** [comment]
Think this would be a great idea and really helpful to understand what the garment is if you're new.

Reminds us of the techdraw/blueprint workspace in FreeCAD but assume that having a web tool would be better so it can be integrated into the site.

Example of use in FreeCAD - https://community.preciousplastic.com/how-to/make-blueprints-in-freecad

Could be that FreeCAD has a workspace for technical drawings for clothes that is open source and could be adapted

**Comment by joostdecock** [comment]
Done. Closing.

**Reply by darigovresearch** [reply]
Thanks!
